docs/cql/dml.rst

```diff
@@ -203,12 +203,7 @@ It is also possible to “group” ``CLUSTERING COLUMNS`` together in a relation
 
 will request all rows that sort after the one having “John's Blog” as ``blog_title`` and '2012-01-01' for ``posted_at``
 in the clustering order. In particular, rows having a ``post_at <= '2012-01-01'`` will be returned as long as their
-``blog_title > 'John''s Blog'``, which would not be the case for::
-
-    SELECT * FROM posts
-     WHERE userid = 'john doe'
-       AND blog_title > 'John''s Blog'
-       AND posted_at > '2012-01-01'
+``blog_title > 'John''s Blog'``.
```

Comment:
Why do you call that example "non working"? Just because an "allow filtering" is missing?

I'm not saying that this is a super-important example, and wouldn't shed too many tears if it is removed, but I think somebody added this example to drive in the point of how tuple inequality is **not** the same as two separate inequalities.

Reply:
IMHO adding ALLOW FILTERING is like cheating in this context, as it bypasses the keys all together.
In any case I do not think this second example is helpful.

Reply:
It's not "cheating" in the sense that the example was trying to show (I am just guessing, I didn't write it), the **meaning** of what does it mean to compare tuples, and how it is not the same as comparing the two elements separately and ANDing it.

The question of whether ALLOW FILTERING is needed or not is an implementation detail of how Scylla implements efficient searching. It's not related to what the comparison means.

Reply:
Initially, I was for leaving both examples, as they helped me understand that if you use the tuple notation `AND (blog_title, posted_at) > ('John''s Blog', '2012-01-01')`, differnt rows will be returned compared to  `AND blog_title > 'John''s Blog'  AND posted_at > '2012-01-01'`.
   
However, if the second example returns an error (see https://github.com/scylladb/scylladb/issues/11719), I think we should remove it because it may be misleading - and probably is, as it was reported by a user.
If I were a user new to ScyllaDB and CQL, I would use both examples to compare the results - and hit the same error.